### DISCUSSION TITLE
Is it possible to have a deployment

### DISCUSSION THREAD

**Discussion by sbmsr** [vercel/vercel#4944] [Help] [answered]
Hi Vercel! Love your product. Have one quick question : 

I would like to have a deployment with non-prod environment variables. It seems development vars are used when running locally, preview vars are used on emphemeral deployments triggered by PR to master, and prod vars are used on deployment corresponding to master branch.

Is there a way to configure a deployment of master, with development or preview env?

**Comment by styfle** [comment, marked as answer]
Hi @r3ticuli 

The default branch you configure with your git provider is assigned to the "production" environment.

You can configure the branch you wish to use for production to something else other than `master` (perhaps `main`), thus `master` will become a "preview" deployment just like your other branches/PRs.

See this blog post: https://vercel.com/blog/custom-production-branch

Does that solve your problem?
